packages/css/src/plugins/expand-shorthands/outline.tsx

```diff
@@ -75,8 +75,8 @@ export const outline: ConversionFunction = (value) => {
   }
 
   return [
-    { prop: 'outline-color', value: colorValue || 'initial' },
-    { prop: 'outline-style', value: styleValue || 'initial' },
-    { prop: 'outline-width', value: widthValue || 'initial' },
+    { prop: 'outline-color', value: colorValue || 'currentColor' },
```

Comment:
Did we intentionally made `colorCode` camel case to fit in [this issue](https://github.com/atlassian-labs/compiled/issues/318)? Because in `text-decoration.tsx` we are setting it as lower case => `{ prop: 'text-decoration-color', value: colorValue || 'currentcolor' }`

Reply:
i noticed it because the spec mentions both `currentColor` and `currentcolor`... and ideally they should be just be one value 🤔 

i'll update the code to be all lower case for now

Reply:
done